### PR TITLE
Tour dependent adjustments: finetuning

### PR DESCRIPTION
**Pull request by TheBestPessimist**
Followup of the review: https://github.com/metasfresh/metasfresh/pull/5945#pullrequestreview-334728891

Task: https://github.com/metasfresh/metasfresh/issues/5885

### CODE REVIEW

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUShipperTransportationBL.java [hunk 1]
```diff
@@ -75,7 +75,7 @@
 		// Make sure Shipper Transportation is still open
 		if (shipperTransportation.isProcessed())
 		{
-			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@");
+			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@", new Object[] { shipperTransportation.getM_ShipperTransportation_ID() });
```
Comment (metas-ts) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUShipperTransportationBL.java]: i think it doesn't work like this..maybe you can clarify with teo..at any rate, i believe the M_ShipperTransportation_ID won't make it into the error message

File: de.metas.handlingunits.base/src/main/sql/postgresql/system/5539950_sys_gh5885_Fix_IsAnonymousHuPickedOnTheFly_Column_name.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+-- 2019-12-20T09:41:59.896Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Element SET ColumnName='IsAnonymousHuPickedOnTheFly',Updated=TO_TIMESTAMP('2019-12-20 11:41:59','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Element_ID=577441
```
Comment (metas-ts) [inline de.metas.handlingunits.base/src/main/sql/postgresql/system/5539950_sys_gh5885_Fix_IsAnonymousHuPickedOnTheFly_Column_name.sql]: See https://github.com/metasfresh/metasfresh/pull/5945#discussion_r363278473

File: de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/M_Tour_Instance_CreateFromSelectedDeliveryDays.java [hunk 3]
```diff
@@ -197,7 +196,7 @@ private I_M_ShipperTransportation getM_ShipperTransportation_Param()
 		// Make sure shipper transportation is not processed
 		if (p_M_ShipperTransportation.isProcessed())
 		{
-			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@");
+			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@", new Object[] { p_M_ShipperTransportation.getM_ShipperTransportation_ID() });
```
Comment (metas-ts) [inline de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/M_Tour_Instance_CreateFromSelectedDeliveryDays.java]: as written further-up; i think thise does not add the M_ShipperTransportation_ID to the error message.
what i had in mind is `....appendParametersToMessage().setParameter("M_ShipperTransportation_ID", p_M_ShipperTransportation.getM_ShipperTransportation_ID())`

File: de.metas.handlingunits.base/src/main/sql/postgresql/system/5539950_sys_gh5885_Fix_IsAnonymousHuPickedOnTheFly_Column_name.sql (hunk 2 above)
Comment (TheBestPessimist) [reply de.metas.handlingunits.base/src/main/sql/postgresql/system/5539950_sys_gh5885_Fix_IsAnonymousHuPickedOnTheFly_Column_name.sql]: field renamed

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUShipperTransportationBL.java (hunk 1 above)
Comment (TheBestPessimist) [reply de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUShipperTransportationBL.java]: fixed

File: de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/M_Tour_Instance_CreateFromSelectedDeliveryDays.java (hunk 3 above)
Comment (TheBestPessimist) [reply de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/M_Tour_Instance_CreateFromSelectedDeliveryDays.java]: fixed

**Pull request merged**